Automattic/jetpack
# VideoPress: Point non iframe player script at latest version

Fixes #

#### Changes proposed in this Pull Request:

* Updates the script used for rendering a non-iframed VideoPress player to point to the latest version of the player.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


Nope

#### Testing instructions:






1. For a site with VideoPress enabled
2. Add the following filter somewhere it will get loaded (like `tools/docker/mu-plugins/debug.php`)
```
function videopress_use_iframe( $use_iframe ) {
    return false;
}

add_filter( 'jetpack_videopress_player_use_iframe', 'videopress_use_iframe', 10, 1 );
```
3. add a video shortcode to a post `[videopress <guid>]`
4. the video player should load and all features work as expected
